### PR TITLE
[Doc] Optimize manager documentation

### PR DESCRIPTION
**Pull request by JNSimba**
# Proposed changes

Issue Number: close #xxx

WIP Doris Manager English Document ~

## Problem Summary:

Optimize manager documentation

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by hf200012 — DISMISSED** [review]
LGTM

File: docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md [hunk 1]
```diff
@@ -27,15 +27,36 @@ under the License.
 # 编译与部署
 
 ## 编译
-直接运行manager路径下的build.sh脚本，会在manager路径下生成安装运行包——output，其中包括:
-1、Doris Manager的运行包doris-manager.jar
-2、运行的配置文件夹conf
-3、启动脚本start_manager.sh
-4、停止脚本stop_manager.sh
+直接在manager路径下的build.sh脚本
```
Comment (EmmyMiao87) [inline docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md:30]: Update English document together ~

Comment (hf200012) [reply docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md:30]: The English documentation has not yet been written

Comment (EmmyMiao87) [reply docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md:30]: WIP is ok

**Review by EmmyMiao87 — DISMISSED** [review]
LGTM

File: docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md (hunk 1 above)
Comment (JNSimba) [reply docs/zh-CN/extending-doris/doris-manager/compiling-deploying.md:30]: I just translated the Chinese document through Google, please review it again

**Pull request closed without merging**